traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -308,8 +308,8 @@ def create_asn(self, query_params=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
-	@api_request('get', 'asns/{asn_id:d}', ('2.0',))
-	def get_asn_by_id(self, asn_id=None):
+	@api_request('get', 'asns', ('2.0',))
+	def get_asn_by_id(self, asn_id=None, query_params=None):
```

Comment:
There's already a `get_asns` that does `GET /asns` with optional query parameters; this method should get deleted instead.